lib/phoenix_live_view/js.ex

```diff
@@ -101,6 +101,38 @@ defmodule Phoenix.LiveView.JS do
   >+</button>
   ```
 
+  ## DOM Selectors
+
+  The client utility commands in this module all take an optional DOM selector
+  using the `:to` option.
+
+  This can be a string for a regular DOM selector such as:
+
+  ```elixir
+  JS.add_class("warmer", to: ".thermo")
+  JS.hide(to: "#modal")
+  JS.show(to: "body a:nth-child(2)")
+  ```
+
+  It is also possible to provide scopes to the DOM selector. The following scopes
+  are available:
+
+   * `{:inner, "selector"}` To target an element within the interacted element.
+   * `{:closest, "selector"}` To target the closest element within the interacted
+   element.
+
+   For example, if building a dropdown component, the button could use the `:inner`
```

Comment:
Note that with this example, if there was any kind of nested dropdown, the nested ones would be targeted as well. For a dropdown probably not that relevant, but when doing dialogs it could be problem.

Reply:
That's a good point, the same rule applies when passing a string as a selector, so perhaps closest is the outlier here?

Reply:
Yeah, maybe it would make sense to mention this. The default and inner variant target multiple elements (-> document/element.querySelectorAll) and closest stops when it finds the first match (-> element.closest). As this is "Phoenix.LiveView.**JS**", we could even link to the docs for those?